package.json

```diff
@@ -4,15 +4,18 @@
   "description": "Create an evergreen web application with no build configuration.",
   "license": "MIT",
   "author": "Owen Buckley <owen@thegreenhouse.io>",
-  "main": "src/index.js",
   "homepage": "https://github.com/ProjectEvergreen/create-evergreen-app#readme",
   "repository": {
     "type": "git",
     "url": "git+https://github.com/ProjectEvergreen/create-evergreen-app.git"
   },
   "keywords": [
     "Web Components",
-    "Lit Element"
+    "Lit Element",
+    "Lit Element",
```

Comment:
Lit Element twice? Did you mean LitElement without the space the second time?

Reply:
ah, meant lit- html